### DISCUSSION TITLE
README.md per ogni sotto cartella --> ?

### DISCUSSION THREAD

**Discussion by cirospat** [ondata/guidaPraticaPubblicazioneCSV#4] [Domande] [answered]
@aborruso 
non capisco perchè ogni file finora costruito si chiama `README.md` dentro la sottocartella di `docs`.

Tra l'altro se clicco sulla penna del progetto (vedi sotto)
![image](https://user-images.githubusercontent.com/3757525/146577630-e043a059-6f0e-4019-a4ff-4b8e53698fe2.png)

mi da errore su Github, (vedi sotto)
![image](https://user-images.githubusercontent.com/3757525/146577772-4a2beaeb-f5fd-49b2-b3d6-4985e8076c25.png)

**Comment by aborruso** [comment, marked as answer]
Caro @cirospat per il tasto edit, c'era un errore (vedi #5 ).

In MkDocs, la home di una sezione si può chiamare indifferentemente o `index.md` o `READM.md`. Sto chiamando così, la pagina principale o unica della sezione. Poi nell'URL non si vede, hai soltanto la radice (come in https://ondata.github.io/guidaPraticaPubblicazioneCSV/introduzione/ in cui non vedi `README`)

**Reply by cirospat** [reply]
ok capito,  grazie @aborruso 

quindi ripeto a me stesso per verificare se ho capito:
per ogni sotto cartella c'è un file `README.md` (file principale), sia che in quella sottocartella ci sia un solo file `MD` sia che ce ne siano più di uno.

Se sbaglio please correggimi

**Reply by aborruso** [reply]
> Se sbaglio please correggimi

Yes, tutto chiaro. Mi spiego con un esempio sulla funzione del readme. Immagina di avere questa struttura:

- cartella
  - nomeFIle1.md
  - nomeFile2.md
  - nomeFile3.md

Gli URL sarebbero:

- `sito.it/cartella/nomeFIle1`
- `sito.it/cartella/nomeFIle2`
- `sito.it/cartella/nomeFIle3`

Se uno di questi file, è quello che fa da home per quella sezione, per cui basta quindi l'URL `sito.it/cartella`, quel file si deve chiamare `README` o `INDEX`.

Sono stato chiaro?

**Reply by cirospat** [reply]
yes of course.
Grazie, avevo capito bene.
